src/command/KeyBindingManager.js

```diff
@@ -410,6 +411,13 @@ define(function (require, exports, module) {
             });
         }
         
+        // Also make sure that the second word of PageUp/PageDown has the first letter in upper case.
+        if (/^Page/.test(key)) {
+            key = key.replace(/(up|down)$/, function (match, p1) {
+                return p1.charAt(0).toUpperCase() + p1.substr(1);
+            });
+        }
```

Comment:
Yes, we do need this since we don't want the case-sensitive search of `_keyNames` list to catch this and reject the case-insensitive-matching `key`. We also need the exactly matching case for the shell code to correctly assign the corresponding accelerator.